README.md

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Update docs and download links to point directly at 3.0.0 / 3.0.0-beta1.

Reply:
Done